index.js

```diff
@@ -43,7 +43,7 @@ if (Array.isArray(type)) {
       }
       break;
     case 2:
-      if (type !== ['atom', 'rss2'] && type !== ['rss2', 'atom']) {
+      if (!(type[0] === 'atom' && type[1] === 'rss2') && !(type[0] === 'rss2' && type[1] === 'atom')) {
```

Comment:
```suggestion
      if (!type.every(t => ['atom', 'rss2'].includes(t))) {
```

Reply:
Shouldn't the two types be different? Because paths can be [generated from types](https://github.com/hexojs/hexo-generator-feed/blob/cd72fad72f483f98d6dc1e36289a9e1b60292101/index.js#L65-L66) which might result in duplicate paths.